### PR TITLE
Fix bad use of dollar sign in listing without mathescape

### LINKED ISSUE #3342: LaTeX source in generated PDF/HTML
In the example in Section 18.8.4 Version Date and Build Information, there is `\ldots$` that makes its way into the generated PDF and HTML. It is in the PR that is supposed to be for the release candidate for 3.6: https://test.openmodelica.org/jenkins/job/ModelicaAssociation/job/ModelicaSpecification/job/PR-3340/1/artifact/MLS.pdf

### PR DESCRIPTION
**Pull request by henrikt-ma**
Fixes #3342.

### CODE REVIEW

File: chapters/annotations.tex [hunk 1]
```diff
@@ -1917,9 +1917,9 @@ \subsection{Version Date and Build Information}\label{version-date-and-build-inf
 \annotationindex{versionDate}\annotationindex{versionBuild}\annotationindex{dateModified}\annotationindex{revisionId}
 
 \begin{example}
-\begin{lstlisting}[language=modelica,mathescape=false]
+\begin{lstlisting}[language=modelica,mathescape=false,escapechar=!]
 package Modelica
-  $\ldots$
+  !$\ldots$!
```
Comment (dietmarw) [inline chapters/annotations.tex:1922]: Just a side note, the mathmode is not really necessary for `\ldots`

Comment (henrikt-ma) [reply chapters/annotations.tex]: Right, why not?  I made the suggested change.  Resolving.

**Review by eshmoylova — APPROVED** [review]
Verified that in the generated by Jenkins PDF they appear as dots.

**Review by HansOlsson — APPROVED** [review]
Looks good.

**Comment by HansOlsson** [comment]
However, will not merge yet due to ongoing vote.

**Pull request merged**